### ISSUE TITLE
Not working with io.js

### ISSUE THREAD

**Issue by mshick** [webhook/webhook#43]
I was trying out io.js today, and found that webhook is not working with it. There may be multiple issues, but the first that crept up was a problem with the firebase.auth() method hanging -- it never reaches the auth callback.

Perhaps this is related to the somewhat out-of-date version of firebase currently in use?

**Comment by snide** [comment]
io.js is brand new. Currently we don't have any plans to support it, though who knows what the future holds. Our firebase auth stuff is locked down due to some changes they made when they released their new auth system a couple months ago. We've been in contact with them and are waiting till they add some features back in.

**Comment by mshick** [comment]
Yeah, understood. Just figured I’d let you know. Theoretically there’s no “supporting” needed, but in practice...

FWIW, with the amount of activity over at io.js I can imagine it really taking off.
